packages/regshot.vm/regshot.vm.nuspec

```
@@ -2,12 +2,13 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>regshot.vm</id>
-    <version>1.9.1.20250219</version>
+    <version>1.9.1.20250506</version>
     <authors>maddes, regshot, xhmikosr</authors>
-    <description>Regshot is a small, free and open-source registry compare utility that allows you to quickly take a snapshot of your registry and then compare it with a second one - done after doing system changes or installing a new software product.</description>
+    <description>Regshot is a small registry compare utility that allows you to take a snapshot of your registry and then compare it with a second one after doing system changes.</description>
```

[nitpick] We can remove some duplication making it nicer to read:
```suggestion
    <description>Regshot is a registry comparison tool for tracking system changes by comparing registry snapshots.</description>
```